pyproject.toml

```diff
@@ -52,6 +52,8 @@ dev = ["pytest",
        "flake8",
        "tox"]
```

Comment:
Let's add `erdantic` to the `dev`, `test` and `docs` extras because it will always be required from now. This will also simplify `tox.ini`.